src/main/java/at/hannibal2/skyhanni/utils/NumberUtil.kt

```diff
@@ -206,4 +206,8 @@ object NumberUtil {
     val Int.milion get() = this * 1_000_000.0
     private val Int.bilion get() = this * 1_000_000_000.0
     val Double.milion get() = (this * 1_000_000.0).toLong()
+
+    fun Number.percentOf(maxValue: Number) = maxValue.toDouble().takeIf { it != 0.0 }?.let { max ->
```

Comment:
Misnomer: percentage is part out of a 100. Maybe `fractionOf` or `progressTowards`?